### ISSUE TITLE
Consider a license change to AGPL.

### ISSUE THREAD

**Issue by nilsding** [dubtrack/www-dubtrack-fm#107]
Looking at the `LICENSE` file in the repo's root, I figured out this project is currently licensed under the GPLv2.  Since this software is designed to run on a server, I recommend you to change it to the [AGPL](http://www.gnu.org/licenses/agpl.html) ([plain text version](http://www.gnu.org/licenses/agpl.txt)).

Now you may ask yourself, ``Why should I use the AGPL for these applications?'' [Because](http://www.gnu.org/licenses/why-affero-gpl.html):

> Suppose you develop and release a free program under the ordinary GNU GPL. If developer D modifies the program and releases it, the GPL requires him to distribute his version under the GPL too. Thus, if you get a copy of his version, you are free to incorporate some or all of his changes into your own version.
> 
> But suppose the program is mainly useful on servers. When D modifies the program, he might very likely run it on his own server and never release copies. Then you would never get a copy of the source code of his version, so you would never have the chance to include his changes in your version. You may not like that outcome.
> 
> Using the GNU Affero GPL avoids that outcome. If D runs his version on a server that everyone can use, you too can use it. Assuming he has followed the license requirement to let the server's users download the source code of his version, you can do so, and then you can incorporate his changes into your version. (If he hasn't followed it, you have your lawyer complain to him.)

**TL;DR** This license requires the person who's running the server application to give anyone who is interested in access to the running source code, regardless of whether it's modified or not.

**Comment by nambok** [comment]
@nilsding thanks for the suggestion, going to ask some people about this. what about the Apache license? Im not the best with licenses this just seem the best option at the time

**Comment by thedark1337** [comment]
According to [this](http://stackoverflow.com/a/6999287) stackOverflow post, the Apache license is far less restrictive 

> Apache 2.0 license doesn't have many restrictions at all. You can distribute modified or derived Apache 2.0 works under other licenses, even proprietary ones. There is no requirement for you to release the source for a hosted service, or even an application distributed to others.

The AGPL however requires you to release the source. imo for an open source website the AGPL license should be better than GPL / Apache.

**Comment by Larry1123** [comment]
Apache 2.0 is my go to. But at this time we can not change the License without @nambok as the code would have to be re-licensed from him.
Also anyone's code that is been pulled in from PRs would have to approve of a License change.
